src/core/public/chrome/ui/loading_indicator.tsx

```diff
@@ -39,16 +39,26 @@ export class LoadingIndicator extends React.Component<LoadingIndicatorProps, { v
     visible: false,
   };
 
+  private timer: any;
+  private increment = 1;
+
   componentDidMount() {
     this.loadingCountSubscription = this.props.loadingCount$.subscribe((count) => {
-      this.setState({
-        visible: count > 0,
-      });
+      if (this.increment > 1) {
```

Comment:
is it the same as `if(this.timer)`?

Reply:
No. See comments above with @pgayvallet.